src/AppBundle/Entity/Todo.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Acme\UserBundle\Entity\User;
+
+/**
+ * Todo
+ *
+ * @ORM\Table(name="todo")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\TodoRepository")
+ */
+class Todo
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="body", type="text")
```

Comment:
As before, i'm trying to avoid having null todos. but then again that could be a bad idea